save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsOrganizationsMenuView.kt

```diff
@@ -56,126 +72,85 @@ class UserSettingsOrganizationsMenuView : UserSettingsView() {
+                                                with(childrenBuilder) {
+                                                    fontAwesomeIcon(icon = faRedo, classes = actionIconClasses.joinToString(" "))
                                                 }
-                                                buttonBuilder("Cancel") {
-                                                    closeWindow()
+                                            }
+                                            classes = actionButtonClasses.joinToString(" ")
+                                            modalButtons = { action, closeWindow, childrenBuilder, _ ->
+                                                with(childrenBuilder) {
+                                                    buttonBuilder(label = "Yes, recover ${organizationDto.name}", style = "danger", classes = "mr-2") {
+                                                        action()
+                                                        closeWindow()
+                                                    }
+                                                    buttonBuilder("Cancel") {
+                                                        closeWindow()
+                                                    }
                                                 }
                                             }
-                                        }
-                                        onActionSuccess = { _ ->
-                                            setState {
-                                                selfDeletedOrganizationWithUserList = selfDeletedOrganizationWithUserList.minusElement(organizationWithUsers)
-                                                selfOrganizationWithUserList = selfOrganizationWithUserList.plusElement(
-                                                    organizationWithUsers.copy(organizationDto.copy(status = OrganizationStatus.CREATED))
-                                                )
+                                            onActionSuccess = { _ ->
+                                                localButtonAction(organizationWithUsers, OrganizationStatus.CREATED)
+                                            }
+                                            conditionClick = false
+                                            sendRequest = { _ ->
+                                                responseChangeOrganizationStatus(organizationDto.name, OrganizationStatus.CREATED)
                                             }
                                         }
-                                        conditionClick = false
-                                        sendRequest = { _ ->
-                                            responseChangeOrganizationStatus(organizationDto.name, OrganizationStatus.CREATED)
-                                        }
+                                        OrganizationStatus.BANNED -> {}
```

Comment:
Is it what you wanted to do?

Reply:
Even if it is,

```suggestion
                                        OrganizationStatus.BANNED -> Unit
```

is a more idiomatic Kotlin.

Reply:
Yes, since only ```SUPER_ADMIN``` can unban an organization, it is illogical that some logic of the organization's unban should be present in the list of the user's personal organizations. Thanks

Reply:
Done